### DISCUSSION TITLE
mono doesn't allow loading it on linux

### DISCUSSION THREAD

**Discussion by FrostKnight** [kwsch/PKHeX#3977] [Q&A]
PS, before you lock this, I am just going to say what others probably should have said.

The fact you immediately lock so no one else can talk in the locked thread and the fact you use c# a really restrictive programming language and only support windows, a garbage infested os that should have been terminated 10+ years ago, really is awful

I am sure there is totally a good reason you are doing this, like for example, having a heart of ice, or just being windows fanboy... and that's about it.  

You would get along well with redhat employees well, they are like linux's  windows 11, bloated and filled with many ideas that are just pointless.

You sadden me, but I won't miss you.

**Comment by kwsch** [comment]
1. Issues are not places to rant. The Discussion space is provided for things that are not issues/improvements with the code project as is.
2. C# is not a "really restrictive programming language".
3. C# runs on Windows, Mac, and Linux. Get with {current_year}.
4. People have figured out how to use the latest versions of PKHeX with [Linux ](https://projectpokemon.org/home/forums/topic/62968-pkhex-on-linux/).
5. PKHeX.Core is available on NuGet. Feel free to write your own GUI with whatever C# GUI framework (Avalonia, MAUI, WPF, Xamarin Forms, etc) or port the entire codebase to your language of choosing (with the same license of course).
6. Considering the last release of PKHeX had 200,000 downloads in a month, I'd say the issue isn't with this repo. It's with you. This is a FREE program, nobody has any obligation to do any work for anyone.
